### PR TITLE
Assignment1-HadiCynthia

### PR DESCRIPTION
**Pull request by HadiM2**
@HadiM2
@montalcy

Site Link: https://mango-tree-002df5010.5.azurestaticapps.net/

### CODE REVIEW

File: website/pages/index.vue
Comment (rahat15) [inline website/pages/index.vue:1]: why do you have a website folder and a wordle web folder?

File: wordle-web/pages/game.vue
Comment (rahat15) [inline wordle-web/pages/game.vue:1]: Good use colors. But why do you have a router here? I did not use one. It works without one though

File: wordle-web/pages/index.vue
Comment (rahat15) [inline wordle-web/pages/index.vue:1]: Good work

File: wordle-web/scripts/letter.ts
Comment (rahat15) [inline wordle-web/scripts/letter.ts:1]: Do you really need this?

File: wordle-web/scripts/word.ts
Comment (rahat15) [inline wordle-web/scripts/word.ts:1]: Same with this. I don't think this is required as well

File: wordle-web/tests/tests.ts
Comment (rahat15) [inline wordle-web/tests/tests.ts:1]: Good work

**Review by rahat15 — COMMENTED** [review]
Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest ✅
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ✅
some default placeholder body text ✅
and a button to take you back to the home page ✅
there should be a button on the home page to access this page ✅
Check all the code into your repo fork ✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅
Extra Credit
Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ❌
One point per idea up to three points

File: website/pages/index.vue
Comment (montalcy) [reply website/pages/index.vue:1]: Forgot to get rid of these, thanks!

File: wordle-web/pages/index.vue
Comment (montalcy) [reply wordle-web/pages/index.vue:1]: thanks!

File: wordle-web/scripts/letter.ts
Comment (montalcy) [reply wordle-web/scripts/letter.ts:1]: No, thanks for the reminder!

**Comment by MikaellaCroskrey-IntelliTect** [comment]
## Assignment 1
## Grading Criteria

1. Create a Vue app ✅
   - Uses TypeScript ✅
   - Add Vuetify ✅
   - Uses Vitest ✅

3. Create a simple page called game.vue ✅
   - Page should have a card with a heading that with a clever name for the game ✅
   - some default placeholder body text ✅
   - and a button to take you back to the home page ✅
   - there should be a button on the home page to access this page ✅

4. Check all the code into your repo fork ✅

5. Create a static web app in Azure ✅
   - Create the container where your GitHub repo will deploy
   - Link the site with your GitHub repo

6. Update the CI/CD pipeline ✅
   - Pipeline runs on each check in
   - Vitest run and pass ✅
   - Vue app builds ✅
   - Vue app deploys to Azure ✅

7. Azure site works ✅
   - Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅

## Extra Credit

- Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ❌
- One point per idea up to three points

> **_NOTE:_** The assignment met the overall acceptance criteria. I made a couple suggestions that are good practices and that should be implemented with future assignments. Overall, great job! :)

**Pull request closed without merging**